<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(225,225,225);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>480</width>
          <height>700</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>170</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>170</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:white;
border-top-right-radius:25px;
border-top-left-radius:25px;</string>
             </property>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>190</x>
                <y>9</y>
                <width>91</width>
                <height>91</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../assets.qrc">:/icons/pfp.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QWidget" name="horizontalLayoutWidget">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>100</y>
                <width>471</width>
                <height>35</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item alignment="Qt::AlignmentFlag::AlignHCenter">
                <widget class="QLabel" name="Name">
                 <property name="styleSheet">
                  <string notr="true">font-size:15pt;
color:black;
font-weight:bold;</string>
                 </property>
                 <property name="text">
                  <string>Mohamed Ramadan</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="horizontalLayoutWidget_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>130</y>
                <width>471</width>
                <height>32</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item alignment="Qt::AlignmentFlag::AlignHCenter">
                <widget class="QLabel" name="PhoneNumber">
                 <property name="styleSheet">
                  <string notr="true">font-size:13pt;
color:gray;</string>
                 </property>
                 <property name="text">
                  <string>+20 11 15921626</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="Bio" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>478</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:white;</string>
             </property>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>25</width>
                <height>25</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:13pt;
color:gray;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../assets.qrc">:/icons/info.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QLabel" name="label_8">
              <property name="geometry">
               <rect>
                <x>49</x>
                <y>6</y>
                <width>114</width>
                <height>33</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:15pt;
color:black;</string>
              </property>
              <property name="text">
               <string>About</string>
              </property>
             </widget>
             <widget class="QLabel" name="About">
              <property name="geometry">
               <rect>
                <x>49</x>
                <y>52</y>
                <width>410</width>
                <height>30</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>410</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>410</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:13pt;
color:gray;</string>
              </property>
              <property name="text">
               <string>3rboka</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>478</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>478</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:white;</string>
             </property>
             <widget class="QLabel" name="commonGroupSectionn">
              <property name="geometry">
               <rect>
                <x>48</x>
                <y>10</y>
                <width>201</width>
                <height>33</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:14pt;
color:black;</string>
              </property>
              <property name="text">
               <string>Common Groups</string>
              </property>
             </widget>
             <widget class="QLabel" name="GroupIcon">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>14</y>
                <width>25</width>
                <height>25</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:13pt;
color:gray;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../assets.qrc">:/icons/group.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QWidget" name="verticalLayoutWidget_3">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>40</y>
                <width>161</width>
                <height>51</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="CummonGroups"/>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:white;
border-bottom-right-radius:25px;
border-bottom-left-radius:25px;</string>
             </property>
             <widget class="QPushButton" name="pushButton">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>10</y>
                <width>151</width>
                <height>48</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size: 11pt;
color: rgb(50,50,50);
background-color:rgb(230,230,230);
border-radius: 11px;
padding:4px;
</string>
              </property>
              <property name="text">
               <string>Delete chat</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/delete.png</normaloff>:/delete.png</iconset>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_3">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>10</y>
                <width>151</width>
                <height>48</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>70</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size: 11pt;
color:white;
background-color:#EF4A4A;
border-radius: 11px;
padding:4px;
</string>
              </property>
              <property name="text">
               <string>Block</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/block.png</normaloff>:/block.png</iconset>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../assets.qrc"/>
 </resources>
 <connections/>
</ui>
